.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,57 @@
+name: Bug Report
+description:  Create a report to help us improve
+labels: ["Type: Bug", "Needs: Triage", "Needs: Lead"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **IMPORTANT: Before posting, remove sensitive data, such as passwords, secret keys, session cookies, etc.**
+
+        Thanks for taking the time to fill out this bug report! 👍
+  - type: textarea
+    attributes:
+      label: Problem
+      description: A short description of the issue you're facing.
+      value: |
+        #### Describe the bug
+
+        #### Evidence / Screenshot
+
+        #### Relevant URL
+
+  - type: textarea
+    attributes:
+      label: Reproducing the bug
+      description: What steps caused you to find the bug?
+      value: |
+        #### Steps to reproduce
+        1. Go to ...
+        2. Do ...
+
+        #### Expected behavior
+
+        #### Actual behavior
+
+  - type: textarea
+    attributes:
+      label: Context
+      description: Additional information to help with debugging.
+      value: |
+        - Browser (Chrome, Safari, Firefox, etc):
+        - OS (Windows, Mac, etc):
+        - Logged in (Y/N): Y
+        - Environment (prod, dev, local): prod
+  - type: textarea
+    attributes:
+      label: Next Steps
+      description: |
+        # Experienced contributors only
+        * What is the proposed solution? Is there a precedent of this approach succeeding elsewhere?
+        * Files related to this issue; this is useful for new contributors who might want to help!
+        * @ tag stakeholders of this bug
+      value: |
+        #### Proposal & constraints
+
+        #### Related files
+
+        #### Stakeholders
```

Comment:
Since this isn't for newcomers, we can keep the comment format on these. Having to mentally `zip` the description with the section is a little annoying. I also tried removing the label if that works.

For the description, also dropped it down to an h2 instead of an h1 to be correct. If that removes the horizontal line, it seems like GitHub's markdown supports `<hr>` so add that before the header :+1: 

```suggestion
      description: |
        ## Experienced contributors only
        Please leave these sections blank for maintainers if you are unsure what to enter!
      value: |
        ### Proposal & constraints
        <!-- What is the proposed solution / implementation? Is there a precedent of this approach succeeding elsewhere? -->

        ### Related files
        <!-- Files related to this issue; this is super useful for new contributors who might want to help! If you're not sure, leave this blank; a maintainer will add them. -->

        ### Stakeholders
        <!-- @ tag stakeholders of this bug -->
```